release-blocking-jobs.md

```diff
@@ -1,88 +1,99 @@
-# Release Blocking Jobs
+# Release Blocking Jobs and Criteria
 
-Here is a list of jobs that should block release if not healthy.
+[SIG-Release TestGrid](https://testgrid.k8s.io/sig-release-master-blocking)
 
-If you want to add a release-branch job, please add them to the appropriate
-sig-release-x.y-all dashboard first. Then show the test is not flaky, and the
-release team can decide if the test should be added to release-blocking
-dashboard as well.
```

Comment:
Seems like [removing this bit] is a policy change (on paper since i can't remember the last time anyone did it this way).
Is there something somewhere that we have to update to stop having the broken *-all dashboards that no one maintains.

You were just discussing these today during the wg-lts and I think you may have sme valuable feedback here as a patch manager.
/cc @tpepper

Reply:
Yeah, this is going to require a follow-up because we currently have no documented procedure for new jobs entering release-blocking.

Reply:
currently we have no real procedure for jobs to be added to informing/blocking.  That's definitely a TODO.